### PR TITLE
header-of-moyo

### PR DESCRIPTION
**Pull request by Prokopovych16**
- [DEMO LINK](https://Prokopovych16.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Prokopovych16.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,8 +7,25 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <a href=""><img src="images/logo.png" alt="" class="img_header"></a>
```
Comment (volodymyr-soltys97) [inline src/index.html:15]:
```suggestion
    <header class="header">
      <a href="">
        <img
          src="images/logo.png"
          alt="Moyo logo"
          class="header__logo"
        >
      </a>
```

File: src/index.html [hunk 2]
```diff
@@ -7,8 +7,25 @@
...
+
+      <nav class="nav">
+        <ul class="ul_header">
+          <li class="li_header"> <a href="" class="a__li">Apple</a></li>
```
Comment (volodymyr-soltys97) [inline src/index.html:19]: Need to fix it everywhere
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="" class="nav__link">Apple</a>
          </li>
          
```

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,59 @@
 body {
+  font-family: 'Roboto';
+  font-size: 12px;
   margin: 0;
+  --height-for-header: 60px;
```
Comment (volodymyr-soltys97) [inline src/style.css:5]: Create a variable with root selector

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,59 @@
...
+}
+a {
+  text-decoration: none;
+  color: black;
+}
+.header {
+  max-width: 100%;
+  height: var(--height-for-header);
+  /* background-color: hotpink; */
```
Comment (volodymyr-soltys97) [inline src/style.css:14]: Remove comments everywhere
```suggestion
```

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,59 @@
...
+  margin-left: 0;
+}
+
+.a__li {
+  position: relative;
+  padding-bottom: 23px;
+}
+
+.a__li:hover {
+  color: #00ACDC;
```
Comment (volodymyr-soltys97) [inline src/style.css:48]: Create a variable for this color and use it everywhere

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Hi, your demo link doesn't work, you need to make a deploy
image

File: src/index.html [hunk 6]
```diff
@@ -7,8 +7,56 @@
...
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+            data-qa="hover"
+            href="#"
+            class="nav__link"
+            >Laptops & Computers</a>
```
Comment (anastasiiavorobiova) [inline src/index.html:44]:
```suggestion
            <a
              data-qa="hover"
              href="#"
              сlass="nav__link"
            >
              Laptops & Computers
            </a>
```
Consider fixing indents according to the code style guide

File: src/index.html [hunk 7]
```diff
@@ -7,8 +7,56 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com">
```
Comment (anastasiiavorobiova) [inline src/index.html:13]:
```suggestion
    <link 
      rel="preconnect"
      href="https://fonts.gstatic.com"
    >
```

File: src/style.css [hunk 8]
```diff
@@ -1,3 +1,66 @@
 body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  text-transform: uppercase;
```
Comment (anastasiiavorobiova) [inline src/style.css:6]:
```suggestion

```
This should be applied just to links

File: src/style.css [hunk 9]
```diff
@@ -1,3 +1,66 @@
...
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__link {
+  position: relative;
+  padding: 23px 0;
+}
+
+.nav__item:nth-child(4) a:hover {
```
Comment (anastasiiavorobiova) [inline src/style.css:49]:
```suggestion
.nav__link:hover {
```

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done! Make sure to deploy your page

**Comment by Prokopovych16** [comment]
I don't understand what is wrong with 'npm run deploy'. It doesn't connect css to html.

**Pull request closed without merging**